.github/workflows/release-public-ami.yml

```diff
@@ -0,0 +1,71 @@
+name: build-public-ami
+
+on:
+  schedule:
+    - cron: '10 10 * * 1'
+  push:
+    tags:
+      - "*"
+
+env:
+  PACKER_VERSION: "1.10.2"
+
+jobs:
+  build-public-ami-and-upload:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 45
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '~1.21.9'
+          check-latest: true
+      - run: go version
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.EXPERIMENTAL_AWS_ACCESS_KEY_PACKER }}
+          aws-secret-access-key: ${{ secrets.EXPERIMENTAL_AWS_SECRET_KEY_PACKER }}
+          #aws-session-token: ${{ secrets.EXPERIMENTAL_AWS_SESSION_TOKEN_PACKER }}
```

Comment:
this might be useful for SSO auth. I used it to test github actions with `act`
I suggets leaving it for now. I am also open to remove it if you feel strong about it